packages/v4/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -0,0 +1,111 @@
+---
+id: Release highlights
+title: Release highlights
+section: get-started
+---
+
+import './get-started.css';
+
+# 2023.06
+[View full details for this release on GitHub](https://github.com/patternfly/patternfly-react/releases).
+
+## Highlights 
+
+To upgrade your codebase to align with this release, [refer to our upgrade guide](/get-started/upgrade).
+
+### Dark theme support
+We are excited to officially support a dark theme on PatternFly. Not only  can you enable dark theme in your products; the PatternFly site can also be now be toggled from light to dark. 
+
+We've updated all of our color variables to ensure that your product is beautiful
+
+There are a couple of ways to enable dark theme in your products. You can: 
+
+1. Add `patternfly-theme-dark.css` to a page and add the `.pf-theme-dark` class to the page's `<html>` element, or 
+2. Apply the `.pf-theme-dark` class to the browser's `prefers-color-scheme` media query. 
+
+### React 18
+We now support the latest version of React, which was released last year. 
+
+### Versioned CSS variables and class names
+We've updated our CSS variables and class names so that they are explicitly associated with the version of PatternFly that they align to. 
+
+For more details, refer to [our upgrade guide](/get-started/upgrade#review-and-update-variable-and-class-names) and our [updated CSS classes and variables](developer-resources/global-css-variables) documentation.
+
+## Most impactful changes
```

Comment:
Are "Most impactful changes" different from "Highlights?"

Reply:
Resolving based on our discussion - I tried renaming this in my last commit,  but we can iterate!